packages/services/api/src/modules/schema/providers/models/composite.ts

```diff
@@ -163,9 +163,9 @@ export class CompositeModel {
     const comparedVersion = compareToPreviousComposableVersion ? latestComposable : latest;
 
     const checksumCheck = await this.checks.checksum({
-      existing: comparedVersion
+      existing: latest
         ? {
-            schemas: comparedVersion.schemas,
+            schemas: latest.schemas,
             contractNames: schemaVersionContractNames,
```

Comment:
let's pass the `contractNames` as part of the latest object, so it is better scoped?